<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-JNHRIVP" failures="1" tests="2" name="TestCases.NegativeTCs" time="250.965" errors="0" timestamp="24 Apr 2021 16:11:12 GMT" skipped="0">
  <testcase classname="TestCases.NegativeTCs" name="verifyTweetmorethan250Char" time="240.994"/>
  <testcase classname="TestCases.NegativeTCs" name="verifyduplicatetweets" time="9.971">
    <failure message="The following asserts failed:
	expected [Something went wrong, but don’t fret — let’s give it another shot.
] but found [his tweet can be longer than 250 character , This tweet can be longer than 250 character , This tweet can be longer than 250 character ,This tweet can be longer than 250 character ,This tweet can be longer than 250 character ,This tweet can be longer than 250 character.Ts tweet caan be longer than 250 character , This tweet can be longer than 250 character , This tweet can be longer than 250 character ,This tweet can be longer than 250 character ,This tweet can be longer than 250 character ,This tweet can be longer than 250 character.
Khaledas
Khaledas
KhaledAshraf.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Something went wrong, but don’t fret — let’s give it another shot.
] but found [his tweet can be longer than 250 character , This tweet can be longer than 250 character , This tweet can be longer than 250 character ,This tweet can be longer than 250 character ,This tweet can be longer than 250 character ,This tweet can be longer than 250 character.Ts tweet caan be longer than 250 character , This tweet can be longer than 250 character , This tweet can be longer than 250 character ,This tweet can be longer than 250 character ,This tweet can be longer than 250 character ,This tweet can be longer than 250 character.
Khaledas
Khaledas
KhaledAshraf.]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at TestCases.NegativeTCs.verifyduplicatetweets(NegativeTCs.java:74)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyduplicatetweets -->
</testsuite> <!-- TestCases.NegativeTCs -->
